Cargo.toml

```diff
@@ -0,0 +1,4 @@
+[workspace]
```

Comment:
Any reason you're using a virtual manifest for this project? It doesn't need it at the moment.

Reply:
Habit. I always anticipate having more projects (e.g. we might want to have more tools here). If you insist on keeping it simple I don't mind removing the workspace.